src/csharp/Microsoft.Spark.E2ETest/UdfTests/BroadcastTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Spark.Sql;
+using static Microsoft.Spark.Sql.Functions;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.UdfTests
```

Comment:
Why is this under `UdfTests`?

Reply:
I thought since they are used in Udfs.. Should I move it under E2ETest directly?

Reply:
It can go to the same level as SparkContextTests.cs.